src/extension.ts

```diff
@@ -191,27 +183,6 @@ async function getDockerInstallationIDHash(): Promise<string> {
     return 'unknown';
 }
 
-/**
- * Workaround for https://github.com/microsoft/vscode/issues/76211 (only necessary if people are on old versions of VS Code that don't have the fix)
- */
-function validateOldPublisher(activateContext: IActionContext): void {
```

Comment:
This didn't need to be removed.

However should we anyway? What do you think @karolz-ms?

Reply:
I'd say let's remove this code; it is ancient history by now.